src/components/personalBankAccountPropTypes.js

```diff
@@ -0,0 +1,55 @@
+import PropTypes from 'prop-types';
+
+export default PropTypes.shape({
+    /** General errors */
+    error: PropTypes.string,
+
+    /** Field specific errors */
+    errorFields: PropTypes.shape({
```

Comment:
I thought errorFields is supposed to be a map of fieldKey to bool and errors a map of timestamp to string?

Reply:
Updated